lib/src/main/java/graphql/nadel/dsl/RemoteArgumentSource.kt

```diff
@@ -1,13 +1,17 @@
 package graphql.nadel.dsl
 
+import graphql.language.Value
+
 // todo this should be a union or sealed class thing
 data class RemoteArgumentSource(
     val argumentName: String?, // for OBJECT_FIELD
     val pathToField: List<String>?,
-    val sourceType: SourceType?,
+    val staticValue: Value<*>?,
```

Comment:
we dont have to do it for this PR - but I think this well become just `astValue` when we do the mix or static or dynamic work later

Reply:
sounds good